src/admin-views/tribe-options-tickets.php

```diff
@@ -60,7 +60,7 @@
 	],
 	'ticket-enabled-post-types' => [
 		'type'            => 'checkbox_list',
-		'label'           => esc_html( sprintf( _x( 'Post types that can have %s', 'tickets fields settings enabled post types', 'event-tickets' ), tribe_get_ticket_label_singular_lowercase( 'tickets_fields_settings_enabled_post_types' ) ) ),
+		'label'           => esc_html( sprintf( _x( 'Post types that can have %s', 'tickets fields settings enabled post types', 'event-tickets' ), tribe_get_ticket_label_plural_lowercase( 'tickets_fields_settings_enabled_post_types' ) ) ),
```

Comment:
```suggestion
		// Translators: %s Tickets, plural.
		'label'           => esc_html( sprintf( _x( 'Post types that can have %s', 'tickets fields settings enabled post types', 'event-tickets' ), tribe_get_ticket_label_plural_lowercase( 'tickets_fields_settings_enabled_post_types' ) ) ),
```

Reply:
done

Reply:
I didn't think this required a changelog entry, but I added that as well.